Source/controls/devices/game_controller.h

```diff
@@ -8,6 +8,14 @@
 
 namespace devilution {
 
+enum GamepadLayout {

```

Comment:
This enum should live in the device agnostic layer, like `controls/game_controls.h`, as it is used in general and not specifically for SDL2 game controllers.